resources/ext/config/conf.d/puppetserver.conf

```diff
@@ -1,14 +1,20 @@
+# Where the puppet-agent dependency places puppet, facter, etc...
+# Puppet server expects to load Puppet from this location
+os-settings: {
```

Comment:
I think this takes us part of the way toward what SERVER-331 was calling for, but not all of the way.  I think we eventually want to just fold the `ruby-load-path` setting back into the `jruby-puppet` section and do away with the `os-settings` section entirely.  @cprice404 - does that sound right?

I'm okay with doing this folding in a follow-up PR for SERVER-331 for the sake of getting this PR through.